github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1127,6 +1127,96 @@ periodics:
         privileged: true
     nodeSelector:
       type: bare-metal-external
+- name: periodic-kubevirt-cluster-sync-test-S390X
+  annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+  cron: "40 4,12,20 * * *"
+  decorate: true
+  cluster: prow-workloads
+  decoration_config:
+    timeout: 4h
+    grace_period: 5m
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+    preset-shared-images: "true"
+    preset-bazel-cache: "true"
+    preset-bazel-unnested: "true"
+    preset-kubevirtci-quay-credential: "true"
+  extra_refs:
+  - org: kubevirt
+    repo: kubevirt
+    base_ref: main
+    work_dir: true
+  - org: kubevirt
+    repo: project-infra
+    base_ref: main
+  reporter_config:
+    slack:
+      job_states_to_report: []
+  spec:
+    nodeSelector:
+      type: bare-metal-external
+    containers:
+      - image: quay.io/kubevirtci/bootstrap:v20240607-febc467
+        env:
+          - name: TARGET
+            value: "external"
+          - name: KUBEVIRT_PROVIDER
+            value: "external"
+          - name: DOCKER_PREFIX
+            value: quay.io/kubevirtci
+          - name: DOCKER_TAG
+            value: "s390x_test"
+          - name: KUBECONFIG
+            value: "/kubeconfig"
+          - name: IMAGE_PULL_POLICY
+            value: "Always"
+          - name: BUILD_ARCH
+            value: "crossbuild-s390x"
+          - name: KUBEVIRT_E2E_FOCUS
+            value: "test_id:4299"
+          - name: GIT_ASKPASS
+            value: "../project-infra/hack/git-askpass.sh"
+        command:
+          - "/usr/local/bin/runner.sh"
+          - "/bin/sh"
+          - "-c"
+          - |
+            # install yq
+            curl -Lo ./yq https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64
+            chmod +x ./yq && mv ./yq /usr/local/bin/yq
+
+            # get kubeconfig
+            source ../project-infra/hack/manage-secrets.sh
+            decrypt_secrets
+            extract_secret 'kubeconfigS390X' /kubeconfig
```

Comment:
this secret doesn't exist  - which external cluster is this job supposed to target?

Reply:
yeah you are correct. Can you please create a secret for the external cluster s390x-prow-virtual cluster and add it with the name kubeconfigs390x. I want this job to target the external 390x-prow-virtual cluster.

Reply:
resolving this conversation as the secret get successfully added with the name `kubeconfigS390X` here.
https://github.com/kubevirt/project-infra/issues/3540